src/main/java/com/gregtechceu/gtceu/integration/ae2/machine/MEInputHatchPartMachine.java

```diff
@@ -117,6 +129,17 @@ public ExportOnlyAEFluidList(MetaMachine machine, int slots, long capacity, IO i
                 this.tanks[i] = new ExportOnlyAEFluid(null, null);
                 this.tanks[i].setOnContentsChanged(this::onContentsChanged);
             }
+            this.fluidStorages = null;
+        }
+
+        @Override
+        public FluidStorage[] getStorages() {
```

Comment:
Could you make the stream creation in `ExportOnlyAEFluidList#handleRecipeInner` use `getStorages()`?

Reply:
ok